test/chi/custom-elements/text-input.pug

```diff
@@ -0,0 +1,251 @@
+---
+title: Text inputs
+---
+script(type='module' src='../../js/ce/ux-chi-ce/ux-chi-ce.esm.js')
+script(nomodule='' src='../../js/ce/ux-chi-ce/ux-chi-ce.js')
+
+each size in ['default', 'sm', 'md', 'lg', 'xl']
+  div(class=`test-${size}`)
+    span.chi-h2= `Text Input / ${size}`
+    table(cellpadding="0", cellspacing="0", width="1200px")
+      each status in ['initial', 'hover', 'focus', 'disabled']
+        tr
+          td
+            small= `${status}`
+          each state in ['default', 'success', 'warning', 'danger']
+            td
+              .-m--1
+                chi-text-input(
+                  state=state=='default'?'':state,
```

Comment:
```suggestion
                  state=state!='default'&&state,
```
Please, check the rest of the file for further occurrences

Reply:
I think Jani's code is ok. This is PUG and not TSX, so maybe, I don't remember well, with Sergio's suggestion it will print "false" when the condition is not met. Jani, please check Sergio's suggestion because it might not work. 
Anyway, please, update the code format to (mind the spaces):
```
state= state === 'default' ? '' : state
```

Reply:
Looks like `state= state != 'default '&& state` this condition syntax is valid for PUG as well, thanks